doc/install/install-ceph-gateway.rst

```diff
@@ -645,3 +617,4 @@ The output should be::
  my-new-bucket
 
 .. _Preflight:  ../../start/quick-start-preflight
+.. _RadosGW frontends site: http://docs.ceph.com/docs/master/radosgw/frontends/
```

Comment:
can't hard-code `master` here, so needs to be a relative url

Reply:
ok, done & thanks for the hint...